examples/probabilistic_matrix_factorization.py

```diff
@@ -0,0 +1,57 @@
+import edward as ed
+from edward.models import Normal
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+
+
+def build_toy_dataset(U, V, N, M, noise_std=0.1):
+  R = np.dot(np.transpose(U), V) + np.random.normal(0, noise_std, size=(N, M))
+  return R
+
+
+def get_indicators(N, M, prob_std=0.5):
+  ind = np.random.binomial(1, prob_std, (N, M))
+  return ind
+
+
+N = 50  # number of users
+M = 60  # number of movies
+D = 3  # number of latent factors
+
+# true latent factors
+U_true = np.random.randn(D, N).astype(np.float32)
+V_true = np.random.randn(D, M).astype(np.float32)
```

Comment:
can you make make these true latent factors larger/smaller? it's hard to tell if inference worked because the prior distributions capture these factors pretty well already.